src/vm/eventreporter.cpp

```diff
@@ -632,6 +632,7 @@ void ReportExceptionStackHelper(OBJECTREF exObj, EventReporter& reporter, SmallS
         OBJECTREF exObj;
         EXCEPTIONREF ex;
         STRINGREF remoteStackTraceString;
+        DebugStackTrace::GetStackFramesData stackFramesData;
```

Comment:
It is not ok to protect the whole `GetStackFramesData` structure. It contains non-GC tracked data like ints or regular unmanaged pointers.

Reply:
This bit is also completely unnecessary; as what I was trying to protect is protected further downstack.

Is a left over of trying to find the potential GC hole